package.json

```diff
@@ -5,9 +5,9 @@
   "description": "Electron application boilerplate based on React, React Router, Webpack, React Hot Loader for rapid application development",
   "main": "main.js",
   "scripts": {
-    "test": "cross-env NODE_ENV=test BABEL_DISABLE_CACHE=1 mocha --retries 2 --compilers js:babel-register --recursive --require ./test/setup.js test/**/*.spec.js",
```

Comment:
I think we should separate unit tests and integration tests, the `e2e.spec.js` should not be in `npm test`, it will affect the experience of `test-watch` (We need `npm run build` first).